src/main/resources/assets/minecolonies/blockstates/blockHutShepherd.json

```diff
@@ -0,0 +1,23 @@
+{
+  "variants": {
+    "facing=up": {
+      "model": "minecolonies:blockhutshepherd"
+    },
+    "facing=east": {
+      "model": "minecolonies:blockhutshepherd",
+      "y": 0
+    },
```

Comment:
can you add the new lines at the end of the files?

Reply:
yes, done